.gitignore

```diff
@@ -1,3 +1,8 @@
-.idea/
-.gradle/
-build/
+/.gradle/
+/.project
+/.settings
+/*/.project
+/*/.classpath
+/*/.settings
+/*/bin
+/*/build
```

Comment:
seems that you just overwrote everything with your own set up. Is it really working? `/` breaks everything on my machine. And you have removed `.idea`. Which is probably good thing. And should also apply for eclipse.